packages/calimero-sdk/src/types/rpc.ts

```diff
@@ -74,6 +76,7 @@ export interface RpcMutateParams<Args> {
   contextId: ContextId;
   method: string;
   argsJson: Args;
+  executorPublicKey: number[];
```

Comment:
Sucks this made it here now (as it should be `bs58`), we could've been able to make this a `string` alias like https://github.com/calimero-network/core/blob/a6ac6526529e60d651ce4c70b51945804f01c364/packages/calimero-sdk/src/types/context.ts

we can't rn, (not trivially anyway), because https://github.com/calimero-network/core/pull/536#discussion_r1698298492 wasn't implemented. we'll have to revisit